src/query/execute_schema.c

```diff
@@ -10502,6 +10502,35 @@ do_change_att_schema_only (PARSER_CONTEXT * parser, DB_CTMPL * ctemplate, PT_NOD
       goto exit;
     }
 
+  /* if it is an auto_increment column, check its domain */
+  if (found_att->auto_increment != NULL)
+    {
+      PT_NODE *att;
+      const char *att_name;
+
+      att = attribute->info.attr_def.attr_name;
```

Comment:
It seems better to put these two lines under the default case below
because att and att_name are not used in other cases.

Reply:
Ok, thanks